config.js

```diff
@@ -1,77 +1,77 @@
 module.exports = {
-  "flags": {
-    "injectTestUrls": false,
-    "itemsExpandable": false,
-    "itemsSwipable": false,
+  'flags': {
```

Comment:
These quotes aren't technically needed at all. The reason they are there is because this used to be a JSON file, then we moves to JS so we could comment stuff out. Perhaps we should just move back to JSON?